composer.json

```diff
@@ -1,10 +1,10 @@
 {
-    "name": "laravel-frontend-presets/zurb-foundation",
-    "description": "Laravel 5.5.x Front-end preset for Zurb Foundation",
+    "name": "crazyinventor/zurb-foundation-laravel-frontend-presets",
```

Comment:
Please keep the vendor name

Reply:
Of course. Sorry, this wasn’t intentional. I changed the vendor/package to test it and forgot to restore the old value before creating a pull request.